index.js

```diff
@@ -167,9 +174,7 @@ class LogstashUDP extends Transport {
     }
 
     try {
-      const logMessage = this._buildLog(info);
-
-      this.sender.send(logMessage, (err) => {
+      this.sender.send(info[MESSAGE], (err) => {
```

Comment:
The `info[MESSAGE]` is the formatted message as built in the `format` phase of the `winston-transport` pipeline. The `info` itself is still the same log object.